src/Be.Vlaanderen.Basisregisters.ProjectionHandling.SqlStreamStore/EnvelopeFactory.cs

```diff
@@ -38,6 +38,37 @@ public object Create(StreamMessage message)
             return envelope;
         }
 
+        public bool TryCreate(StreamMessage message, out object envelope)
+        {
+            if (_eventMapping.HasEventType(message.Type))
+            {
+                var @event = Deserialize(message);
```

Comment:
Can we replace this entire chunk by `envelope = Create(message);`? To not duplicate the same thing right above

Reply:
Done